gk/main.c

```diff
@@ -157,17 +154,33 @@ priority_from_delta_time(uint64_t present, uint64_t past)
 	return integer_log_base_2(delta_time);
 }
 
+static int
+get_grantor_id(struct gk_lpm *ltbl, struct ip_flow *flow)
```

Comment:
Currently, there are two calls to ``get_grantor_id()``, one is ``add_new_flow_from_policy()``, which needs to test if the FIB entry has action ``GK_FWD_GRANTOR``; another one is ``gk_proc()`` when the flow entry is not in the LPM table, I think the FIB entry there doesn't necessarily have an action ``GK_FWD_GRANTOR``, since it may instruct to drop the packet. Any thoughts?